lib/phonebook/providers/association_member_list_provider.dart

```diff
@@ -16,10 +18,13 @@ class AssociationMemberListNotifier extends ListNotifier<CompleteMember> {
   }
 
   Future<AsyncValue<List<CompleteMember>>> loadMembers(
-      String associationId, String year) async {
+      String associationId, String year, ref) async {
     final result = await loadList(() async => associationMemberRepository
         .getAssociationMemberList(associationId, year));
     debugPrint("loadMembers: $result");
```

Comment:
Take care of this debug print when you will mark this PR as ready to be reviewed